docs/using.rst

```diff
@@ -179,13 +179,9 @@ want to use the Apache plugin, it has to be installed separately:
    emerge -av app-crypt/letsencrypt
    emerge -av app-crypt/letsencrypt-apache
 
-Currently, only the Apache plugin is included in Portage. However, if you
-Warning!
-You can use Layman to add the mrueg overlay which does include a package for the
-Certbot Nginx plugin, however, this plugin is known to be buggy and should only
-be used with caution after creating a backup up your Nginx configuration.
-We strongly recommend you use the app-crypt/letsencrypt package instead until
-the Nginx plugin is ready.
+Currently, only the Apache plugin is included in Portage. However, you
+can use Layman to add the mrueg overlay which does include a package for the
+Certbot Nginx plugin.
```

Comment:
I'm not sure we want to make this change. Unless you have information about it that I don't, we have very little about the `letsencrypt-nginx` package included in the mrueg overlay. Has the package been renamed to Certbot? Is it actively maintained? We removed all instructions about doing this from the instruction generator before we launched. I think we should consider doing the same here.